cmd/runtime_options.go

```diff
@@ -95,6 +107,32 @@ func getRuntimeOptions(flags *pflag.FlagSet, environment map[string]string) (lib
 		}
 	}
 
+	if !opts.NoThresholds.Valid { // If not explicitly set via CLI flags, look for an environment variable
+		if envVar, ok := environment["K6_NO_THRESHOLDS"]; ok {
+			val, err := strconv.ParseBool(envVar)
+			if err != nil {
+				return opts, err
+			}
+			opts.NoThresholds = null.BoolFrom(val)
+		}
+	}
+
+	if !opts.NoSummary.Valid { // If not explicitly set via CLI flags, look for an environment variable
+		if envVar, ok := environment["K6_NO_SUMMARY"]; ok {
+			val, err := strconv.ParseBool(envVar)
+			if err != nil {
+				return opts, err
+			}
+			opts.NoSummary = null.BoolFrom(val)
+		}
+	}
+
+	if !opts.SummaryExport.Valid { // If not explicitly set via CLI flags, look for an environment variable
+		if envVar, ok := environment["K6_SUMMARY_EXPORT"]; ok {
+			opts.SummaryExport = null.StringFrom(envVar)
+		}
+	}
+
```

Comment:
See https://github.com/loadimpact/k6/pull/1599/files#r482818305

Reply:
I fixed this in https://github.com/loadimpact/k6/pull/1788/commits/f12d7b72463e6fe1b3f6a516944dc1daec2cee8f